lib/CleantalkSP/SpbctWP/Upgrader.php

```diff
@@ -91,6 +91,7 @@ public function run( $options ) {
 		$options = wp_parse_args( $options, $defaults );
 		
 		$options = apply_filters( 'upgrader_package_options', $options );
+		$options = apply_filters( 'spbct_upgrader_package_options', $options );
```

Comment:
Need to replace this hook with something like "spbct_upgrader_package_url" filtered the URL only, not all options.

Reply:
Done, but in the other way, please, look at the new commits.